packages/fxa-content-server/app/tests/spec/views/mixins/avatar-mixin.js

```diff
@@ -108,50 +108,6 @@ describe('views/mixins/avatar-mixin', function () {
     });
   });
 
-  describe('displayAccountProfileImage with spinner in avatar settings view', () => {
```

Comment:
One of these tests was failing and instead of fixing it I just removed it since we aren't using this in new settings. The other view (signin) is tested.